hamming/hamming.py

```diff
@@ -0,0 +1,12 @@
+"""Calculate the Hamming difference between two DNA strands."""
+
+
+def hamming(strand1, strand2):
+
+    count = 0
+    if len(strand1) != len(strand2):
```

Comment:
What I would suggest is having a guard clause at the start, before any other logic:

```
def hamming(strand1, strand2):
    if len(strand1) != len(strand2):
        raise ArgumentError("The lists are different lengths")

    if strand1 == "" or strand2 == "":
         raise ArgumentError("One of the strands are empty.")
```

Note `ArgumentError` is more appropriate and you need an `or` on line 9.